src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.props

```diff
@@ -61,14 +66,21 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!--
     Configures the file extension used for generated C# files.
     -->
-    <RazorGenerateOutputFileExtension>.g.cshtml.cs</RazorGenerateOutputFileExtension>
+    <RazorGenerateOutputFileExtension>.g.cs</RazorGenerateOutputFileExtension>
```

Comment:
Why change this? We did the `.g.cshtml.cs` on purpose (after Blazor SDK was written)

Reply:
Ok I forgot why I did this. I was planning on changing the way we name the generated file. Today we change extension, so you get: `Index.cshtml` -> `Index.g.cshtml.cs`. This is a bit crummy for `.razor` files: `Index.razor` -> `Index.g.cshtml.cs`. My plan was to append this value rather than change the extension:

* `Index.cshtml` -> `Index.cshtml.g.cs`
* `Index.razor` -> `Index.razor.g.cs`
* `Index.razor` -> `Index.razor.g.i.cs` (definition file)

Reply:
OK